Formula/libplctag.rb

```diff
@@ -0,0 +1,16 @@
+class Libplctag < Formula
+  desc "Portable and simple API for accessing AB PLC data over Ethernet"
+  homepage "https://github.com/kyle-github/libplctag"
+  url "https://github.com/kyle-github/libplctag/archive/v1.5.8.tar.gz"
+  sha256 "03847aeb4bdbfad255e62feff029b9b0c81758e66b840237023243b1c1b764ec"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```

Comment:
Hum… no

Reply:
We need a test that exercises the some of the functionality of the app, or that the library is well compiled. You could write, compile and link a simple C program (see the `mpfr` formula as example).

Reply:
ok, thank you, I'll follow the example!